packages/oidc-dpop-client-browser/package.json

```diff
@@ -13,7 +13,8 @@
     "clean": "npm run clean-module && npm run clean-browser && npm run clean-examples && rm -rf ./coverage",
     "clean-module": "rm -rf ./dist",
     "clean-browser": "rm -rf ./browserDist",
-    "build": "rollup --config rollup.config.js",
+    "rollup": "rollup --config rollup.config.js",
+    "build": "tsc -p tsconfig.json",
```

Comment:
Just a niggle, but I prefer full switches, so here it'd be "tsc --project tsconfig.json".

Reply:
Since Vincent figured out an issue with the rollup config, the pure `tsc` script is no longer required. Resolved in 04c8dfbdcd27bfe611a839d92656fd2f8d915fd5.